README.md

```diff
@@ -344,6 +344,72 @@ const Title = glamorous.h1({
 </ThemeProvider>
 ```
 
+`glamorous` also exports a `withTheme` higher order component (HOC) so you can access your theme in any component!
+> Try this out in your browser [here](https://codesandbox.io/s/qYmJjE4jy)!
```

Comment:
Very cool! When this gets merged and released, you'll need to update the glamorous version in your packages on that codesandbox :)